tests/helpers/vars.js

```diff
@@ -0,0 +1,36 @@
+const unlockPort = process.env.UNLOCK_PORT || 3000
```

Comment:
👍 on this, makes a lot of sense to avoid duplicating the env var logic everywhere they get pulled in